content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,255 @@
+$ cd get-started-experiments
+$ virtualenv .venv
+$ . .venv/bin/activate
+$ python -m pip install -r requirements.txt
+```
+
+### ✅ Get the data set
+
+The repository we cloned doesn't contain the dataset. We use `dvc pull` to
+update the missing data files.
+
+```dvc
+$ dvc pull
+```
+
+The repository already contains the necessary configuration to run the
+experiments.
+
+## 💡 Preparing a project for DVC experiments
+
+If DVC is not installed to the system, please refer to [install](/doc/install).
+
+In this getting started page, we assume that there is already a configured DVC
+project to simplify the introduction. DVC experiments require a DVC pipeline
+defined in the project.
+
+If DVC is not initialized before in the project, you can do so by:
+
+```dvc
+$ dvc init
+```
+
+DVC also requires commands to be run and their dependencies to be defined as
+stages. We use `dvc stage add` to add a stage and set its dependencies.
+
+```dvc
+$ dvc stage add -n train \
+                -p model.conv_units \
+                -p train.epochs \
+                -d data/images \
+                -m metrics.json \
+                python3 src/train.py
+```
+
+The command tells DVC to create an experiment named `train`, and for any change
+in `data/images/` directory, `model.conv_units` or `train.epochs` parameters, we
+(re)run an experiment using `src/train.py` that produces a new `metrics.json`
+file.
+
+You can learn more about [pipelines], and [parameters] in other sections of the
+documentation.
+
+[pipelines]: /doc/start/data-pipelines
+[parameters]: /doc/start/metrics-parameters-plots
+
+</details>
+
+## 👟 Running the experiment with default parameters
+
+The purpose of `dvc exp` subcommands is to run the experiments without
+committing parameter and dependency changes to Git. The artifacts like models,
+metrics produced by each experiment are tracked by DVC and persisted on demand.
+
+Running the experiment with the default project settings requires only the
+command:
+
+```dvc
+$ dvc exp run
+...
+Reproduced experiment(s): exp-b28f0
+Experiment results have been applied to your workspace.
+...
+```
+
+It runs the specified command (`python train.py`) in `dvc.yaml`. That command
+writes the metrics values to `metrics.json`.
+
+This experiment is then associated with the values found in parameters file
+(`params.yaml`), and other dependencies (`data/images/`) with these produced
+metrics.
+
+<details>
+
+### 📖 If you used `dvc repro` before
+
+Earlier versions of DVC uses `dvc repro` to run the pipeline. If you already
+have a DVC project, you may already be using `dvc repro`.
+
+In DVC 2.0 `dvc exp run` supersedes `dvc repro`. We use `dvc repro` to run the
+pipeline as found in the <abbr>workspace</abbr>. All the parameters and
+dependencies are retrieved from the current workspace. It doesn't use any
+special objects to track the experiments or associate parameters with metrics.
```

Comment:
I don't fully understand the 2nd-4th sentences of this paragraph and which ones refer to `repro` vs `exp run` behavior. I'm not sure they are needed, since the 5th sentence onwards makes clear to me the rationale for using `exp run`.

Reply:
Removed the first sentence that may be causing the confusion.